cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/main.go

```diff
@@ -35,8 +46,5 @@ func RegisterHTTPHandlerFromEndpoint(ctx context.Context, mux *runtime.ServeMux,
 
 // GetPluginDetail returns a core.plugins.Plugin describing itself.
 func GetPluginDetail() *plugins.Plugin {
-	return &plugins.Plugin{
-		Name:    "kapp_controller.packages",
-		Version: "v1alpha1",
-	}
+	return &pluginDetail
```

Comment:
This change really belongs in the next PR where I'll need the plugin detail for each item when returning package repositories from multiple plugins.